Automattic/pocket-casts-ios
# [Kids Profile] Add Kids Profile banner

| 📘 Part of: #1935
|:---:|

Fixes #1936

This PR adds the Kids Profile banner to the Profile View. The feature is behind FF.

| Light | Dark |
| -------- | ------- |
| ![RocketSim_Screenshot_iPhone_15_Pro_6 1_2024-07-25_16 42 21](https://github.com/user-attachments/assets/f19a0659-3ffc-46c4-8fa5-7669b616a66c)  | ![RocketSim_Screenshot_iPhone_15_Pro_6 1_2024-07-25_16 44 02](https://github.com/user-attachments/assets/46e0111a-2b52-4587-9ea5-a878b7cc5640) |


## To test

> [!NOTE]
> You need to enable the `kidsProfile` Feature Flag

- Run the App and enable the Feature Flag
- You should see the banner appearing as first row below the header (see screenshot)
- Confirm that tapping the row won't do anything
- Tap the _x_ button (top right in the banner)
- Confirm the banner disappear (for now if the table reloads the banner will show up again)
- Confirm that switching off the FF will hide the banner

## Checklist
